Description: WebRTC includes both <linux/prctl.h> and <sys/prctl.h>, whose definitions clash under musl (struct prctl_mm_map): the kernel header only on glibc.
Forwarded: not-needed (copium's; its author sends what fits upstream)
Origin: copium 155.2 (https://codeberg.org/selfisekai/copium: the musl patches Alpine's chromium applies), cr140-musl-prctl.patch
Changed from copium's: its '+++' line lacks the '/' after third_party/webrtc (GNU patch takes the '---' name; Jam OS's applier takes the '+++' one).
The author's e-mail address is left out of the From line below.

From 98338d64fff786f366b209c14735ff73b2c353bd Mon Sep 17 00:00:00 2001
From: LN Liberda
Date: Tue, 2 Sep 2025 23:49:49 +0200
Subject: [PATCH] musl/linux: Don't import conflicting libc and kernel headers

/usr/include/sys/prctl.h:88:8: error: redefinition of 'prctl_mm_map'
/usr/include/linux/prctl.h:134:8: note: previous definition is here
---
 rtc_base/platform_thread_types.cc | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/third_party/webrtc/rtc_base/platform_thread_types.cc b/third_party/webrtcrtc_base/platform_thread_types.cc
index 20bf4afc44..5b26c120e4 100644
--- a/third_party/webrtc/rtc_base/platform_thread_types.cc
+++ b/third_party/webrtc/rtc_base/platform_thread_types.cc
@@ -12,7 +12,9 @@
 
 // IWYU pragma: begin_keep
 #if defined(WEBRTC_LINUX)
+#if defined(__GLIBC__)
 #include <linux/prctl.h>
+#endif
 #include <sys/prctl.h>
 #include <sys/syscall.h>
 
